src/main/java/org/kohsuke/github/AbstractBuilder.java

```diff
@@ -42,7 +41,7 @@
  *            Intermediate return type for this builder returned by calls to {@link #with(String, Object)}. If {@link S}
  *            the same as {@link R}, this builder will commit changes after each call to {@link #with(String, Object)}.
  */
-abstract class AbstractBuilder<R, S> extends GitHubInteractiveObject {
+public abstract class AbstractBuilder<R, S> extends GitHubInteractiveObject {
```

Comment:
I don't want to make this class public, but `done()` is inherited and exposed so it needs to be public. 

Need to think on this.